bundles/config/org.eclipse.smarthome.config.discovery.usbserial.linuxsysfs/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse SmartHome Configuration USB-Serial Discovery Linux sysfs Scanning
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.eclipse.smarthome.config.discovery.usbserial.linuxsysfs
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.10.0.qualifier
+Import-Package: 
+ com.google.common.collect,

```

Comment:
avoid

Reply:
I will remove this import and adapt the test code accordingly.